custom_components/tahoma/climate_devices/atlantic_heat_recovery_ventilation.py

```diff
@@ -23,29 +19,42 @@
 from ..coordinator import TahomaDataUpdateCoordinator
 from ..tahoma_entity import TahomaEntity
 
-FAN_BOOST = "boost"
-FAN_AWAY = "away"
+FAN_BOOST = "Home Boost"
+FAN_KITCHEN = "Kitchen Boost"
+FAN_AWAY = "Away"
+FAN_BYPASS = "Bypass Boost"
 
-PRESET_PROG = "prog"
-PRESET_MANUAL = "manual"
+PRESET_AUTO = "Auto"
+PRESET_PROG = "Prog"
+PRESET_MANUAL = "Manual"
 
 COMMAND_SET_AIR_DEMAND_MODE = "setAirDemandMode"
 COMMAND_SET_VENTILATION_CONFIGURATION_MODE = "setVentilationConfigurationMode"
 COMMAND_SET_VENTILATION_MODE = "setVentilationMode"
+COMMAND_REFRESH_VENTILATION_STATE = "refreshVentilationState"
+COMMAND_REFRESH_VENTILATION_CONFIGURATION_MODE = "refreshVentilationConfigurationMode"
 
 IO_AIR_DEMAND_MODE_STATE = "io:AirDemandModeState"
 
 FAN_MODE_TO_TAHOMA = {
     FAN_AUTO: "auto",
     FAN_BOOST: "boost",
-    FAN_HIGH: "high",
+    FAN_KITCHEN: "high",
     FAN_AWAY: "away",
 }
 
 TAHOMA_TO_FAN_MODE = {v: k for k, v in FAN_MODE_TO_TAHOMA.items()}
 
-HVAC_MODES = [HVAC_MODE_COOL, HVAC_MODE_HEAT]
-PRESET_MODES = [PRESET_COMFORT, PRESET_ECO, PRESET_PROG, PRESET_MANUAL]
+HVAC_MODES = [HVAC_MODE_FAN_ONLY]
+PRESET_MODES = [PRESET_AUTO, PRESET_PROG, PRESET_MANUAL]
+FAN_MODES = [FAN_AUTO, FAN_BOOST, FAN_KITCHEN, FAN_AWAY, FAN_BYPASS]
+
+date=date.today()
+DAY=date.day
+MONTH=date.month
+
+VENTILATION_MODE_STATE = {"dayNight": "night","month": MONTH,"test": "off","endOfLineTest": "off","leapYear": "off","day": DAY,"cooling": "off","prog": "off"}
```

Comment:
This won't work, since this is only loaded on first init I believe. Which means that the DAY and MONTH will be hardcoded to the day that you did restart Home Assistant.